packages/build/webpack/app.view.template.xml

```diff
@@ -1,5 +1,5 @@
 <% const { files, options } = htmlWebpackPlugin; %>
-<view xmlns="http://labkey.org/data/xml/view"
+<view xmlns="http://labkey.org/data/xml/view" isAppView="true"
```

Comment:
I believe you'll want to conditionalize this based on `options.viewTemplate` having a value of `app`

Reply:
Good catch! I believe I have resolved the issue.